Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveM.cpp

```diff
@@ -179,7 +196,23 @@ void FiniteDifferenceSolver::MacroscopicEvolveM (
               M_yface(i, j, k, 2) += dt * (-PhysConst::mu0 * mag_gamma_interp) * ( M_yface(i, j, k, 0) * Hy_eff - M_yface(i, j, k, 1) * Hx_eff)
                 + dt * Gil_damp * ( M_yface(i, j, k, 0) * ( M_yface(i, j, k, 2) * Hx_eff - M_yface(i, j, k, 0) * Hz_eff)
                 - M_yface(i, j, k, 1) * ( M_yface(i, j, k, 1) * Hz_eff - M_yface(i, j, k, 2) * Hy_eff));
- 
+
+              // temporary normalized magnitude of M_yface field at the fixed point
+              // re-investigate the way we do Ms interp, in case we encounter the case where Ms changes across two adjacent cells that you are doing interp
+              Real M_normalized = sqrt( pow(M_yface(i, j, k, 0),2.0) + pow(M_yface(i, j, k, 1),2.0) + pow(M_yface(i, j, k, 2),2.0) ) /
```

Comment:
Same comment about amrex::  and std:: operators

Reply:
Sure!